### ISSUE TITLE
Translations are not loading

### ISSUE THREAD

**Issue by Azbesciak** [ngneat/transloco#238]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior
Translations are not fetched. Only after usage of pipe/directive. Nevertheless, even then neither `translate` nor `service.translate` functions do not translate keys.

## Expected behavior
Translations are loaded

## Minimal reproduction of the problem with instructions
I just made as you wrote in getting started - ng add etc.

## Environment
Angular version: 8.2.14
translaco latest version (`2.13.3`)

Browser:
- [x] Chrome (desktop) version 80.0.3987.106

**Comment by shaharkazaz** [comment]
Duplicate of #127

**Comment by itayod** [comment]
@Azbesciak you probably didn't wait for the translation to load. 
The `translate` method is translating synchronously as mentioned in the docs!

Try to use `selectTranslate` from the service and see if it works for you, see in [docs](https://netbasal.gitbook.io/transloco/service-api/translation-api#selecttranslate).

**Comment by Azbesciak** [comment]
Yes, thank you. Sorry for the noise.
